lua/location_utils.lua

```diff
@@ -28,7 +28,8 @@ M.telescope_list_or_jump = function(title, params, locations, lsp_client, opts)
       end
     end
 
-    vim.lsp.util.jump_to_location(locations[1], lsp_client.offset_encoding, opts.reuse_win)
+    local show_document = vim.lsp.util.show_document or vim.lsp.util.jump_to_location
+    show_document(locations[1], lsp_client.offset_encoding, opts.reuse_win)
```

Comment:
looking at the docs, `show_document` 3rd parameter is a table with opts, so this should be something like `{ reuse_win = opts.reuse_win, focus = true }`. But if we don't have `show_document`, when old calling convention should be used.

Reply:
Thanks for pointing out that oversight, I see it's a table from the docs.